.travis.yml

```diff
@@ -18,4 +18,4 @@ script:
     - npm test
 
 after_success:
-    - bash <(curl -s https://codecov.io/bash) -s .coverage
+    - test $TRAVIS_BRANCH = "master" && bash <(curl -s https://codecov.io/bash) -s .coverage
```

Comment:
Should change this to check for `CODE_COV_TOKEN` later. But it works for our purposes meanwhile.